hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/ApiBoundaryTypeAttributeEnricher.java

```diff
@@ -35,11 +35,20 @@
       EnrichedSpanConstants.getValue(BoundaryTypeValue.BOUNDARY_TYPE_VALUE_ENTRY);
   private static final String EXIT_BOUNDARY_TYPE =
       EnrichedSpanConstants.getValue(BoundaryTypeValue.BOUNDARY_TYPE_VALUE_EXIT);
+  private static final String X_FORWARDED_HOST_SERVER = "x-forwarded-server";
```

Comment:
I don't see the need for adding these to raw constants and we're anyways planning to convert them into Java constants, hence added them here directly.